### PR TITLE
[a11y] Fix missing aria-label in pagenavigation

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue # .

### Summary of Changes
The <nav in pagenavigation needs an aria-label. 

### Testing Instructions
You need a site where you have a pageanavigation.
Inspect the sourceode before and after patch 

### Actual result BEFORE applying this Pull Request
No aria-label for the <nav element

### Expected result AFTER applying this Pull Request
grafik

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: plugins/content/pagenavigation/tmpl/default.php [hunk 1]
```diff
@@ -13,10 +13,12 @@
 use Joomla\CMS\Language\Text;
 use Joomla\CMS\Router\Route;
 
+$this->loadLanguage();
+
 $lang = $this->getLanguage();
 ?>
 
-<nav class="pagenavigation">
+<nav class="pagenavigation"  aria-label="<?php echo Text::_('PLG_PAGENAVIGATION_ARIA_LABEL'); ?>">
```
Comment (brianteeman) [inline plugins/content/pagenavigation/tmpl/default.php]:
```suggestion
<nav class="pagenavigation" aria-label="<?php echo Text::_('PLG_PAGENAVIGATION_ARIA_LABEL'); ?>">
```

remove extra space

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on aab5d6544e42b9acdf698ec78d5a81f651c5c820<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43600">issues.joomla.org/tracker/joomla-cms/43600</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on aab5d6544e42b9acdf698ec78d5a81f651c5c820<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43600">issues.joomla.org/tracker/joomla-cms/43600</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43600">issues.joomla.org/tracker/joomla-cms/43600</a>.</sub>

**Comment by MacJoom** [comment]
Thank you!

**Pull request merged**